README.md

```diff
@@ -108,7 +155,7 @@ NimbleTemplate supports 4 variants:
 
 #### 2.2/ Phoenix project
 
-The Phoenix project could be either a Web or API project.
+The Phoenix project could be either a **Web** or **API** project.
```

Comment:
@byhbt Could you help to add LiveView in here too 🙏

```
The Phoenix project could be either a **Web**, **LiveView** or **API** project.
```

Reply:
Updated!